src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -54,17 +68,28 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   const containerCssVars = useCss({
     "--event--margin-left": `${eventMarginLeftPx}px`,
     "--event--width": `${eventWidthPx}px`,
+    "--event--expanded-width": `${expandedEventPx}px`,
   });
 
+  const isEventLong = event.duration_minutes >= SCHEDULE_LONG_EVENT_LENGTH_MIN;
+
   const containerClasses = classNames(
     "ScheduleEvent",
     {
       "ScheduleEvent--live": isEventLive(event),
       "ScheduleEvent--users": isPersonalizedEvent,
+      "ScheduleEvent--short": !isEventLong,
     },
     containerCssVars
   );
 
+  const expandClasses = classNames("ScheduleEvent__expand", {
+    "ScheduleEvent__expand--hidden": isEventLong,
+    "ScheduleEvent__expand--marged": !isEventLong,
+    "ScheduleEvent__expand--padded": isEventLong,
```

Comment:
there are 2 classnames that are added when `isEventLong` is true. can we replace it by one of them. and then don't forget to update the styles.

Reply:
in fact, I think we need here only `--hidden`. and move `--marged` styles (margin: auto)  to the `ScheduleEvent__expand`

Reply:
fixed these classnames 64c0c27